### DISCUSSION TITLE
Workspace width adjustment

### DISCUSSION THREAD

**Discussion by PolyShang** [SlRvb/Obsidian--ITS-Theme#130] [Q&A] [answered]
I am using the latest ITS theme and love it, though I hoped to be able to adjust the workspace width and didn't figure out how to do it with ITS.

Basically, I liked to expand the workspace width (the default is too narrow for me) but don't want to go full-blown to occupy all of the available space. Is there a way I could do it with ITS?

Or, if the theme doesn't support it, perhaps someone would add the feature in future releases, or point me where to find a solution?

**Comment by SlRvb** [comment, marked as answer]
In the Style Settings, the option you're looking for is under: `Workspace > Workspace Resize > Note Width`

**Reply by PolyShang** [reply]
Thanks a lot, and got it done thanks to your help.
